devtools-gac.bat

```diff
@@ -0,0 +1,12 @@
+﻿@echo off
```

Comment:
There's a nuke target for doing this - I don't think we should be adding more script files to the root of the directory
`./build.ps1 GacAdd -Framework net461`

Reply:
fixed in 46c5212